client/src/components/ReviewModal.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Rating from '@mui/material/Rating';
+import TextField from '@mui/material/TextField';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Modal from '@mui/material/Modal';
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
+  border: '2px solid #000',
+  boxShadow: 24,
+  p: 4,
+};
+
+export default function ReviewModal({ handleClose, open, profile }) {
+  const [value, setValue] = React.useState(0);
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+    const feedback = data.get('feedback');
+    const rating = data.get('half-rating');
+    const rider = data.get('wasRider');
+    const driver = data.get('wasDriver');
+    const date = new Date();
+    console.log(feedback, rating, rider, driver, date);
```

Comment:
I suggest you deleting this console.log.

Reply:
This is for them to know what they are expecting when a post request is added